landing/src/components/modules/bridge-out-step-2.tsx

```
@@ -86,23 +86,27 @@ export const BridgeOutStep2 = () => {
+  const updateResourcesFromSelectedDonkeys = useMemo(
+    () => (selectedDonkeyIds: Set<bigint>) => {
+      const allResources = Array.from(selectedDonkeyIds).flatMap(
+        (id) =>
+          donkeyInfos?.find((d) => d?.donkeyEntityId && BigInt(d.donkeyEntityId) === id)?.donkeyResourceBalances || [],
+      );
+
+      setSelectedResourceIds(allResources.map((r) => r.resourceId as never));
+      setSelectedResourceAmounts(
+        allResources.reduce(
+          (acc, r) => ({
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)